### DISCUSSION TITLE
PSRule.Rules.Azure/customization/enforce-codeowners/

### DISCUSSION THREAD

**Discussion by giscus** [Azure/PSRule.Rules.Azure#2510] [Documentation]
# PSRule.Rules.Azure/customization/enforce-codeowners/

Learn how to create a custom rule to validate the file path of code artifacts.

https://azure.github.io/PSRule.Rules.Azure/customization/enforce-codeowners/

**Comment by piaccarino** [comment]
People have been asking for feature parity with another product you own, not to mention GitLab as well, with a simple CODEOWNERS and you offer... this?

**Reply by BernieWhite** [reply]
We don't have any feature requests for anything related to code ownership on this repo currently. Feel free to add one so we can understand your use case.

**Reply by g0pinath** [reply]
The use case is - pretty much the whole world is using it.

**Reply by BernieWhite** [reply]
Hi @g0pinath are you able to provide additional details on how you would like static analysis testing of your Azure Bicep code or ARM templates to apply to code ownership?

In this topic we are exploring how the content of files based on static analysis can be used to enhance existing code ownership models.

For general code ownership on GitHub and Azure DevOps you can provide feedback and suggestions here:

- https://devblogs.microsoft.com/devops/azure-devops-feature-suggestions/
- https://github.com/community/community
